receiver/qubes-video-companion

```diff
@@ -4,18 +4,44 @@
 # Licensed under the MIT License. See LICENSE file for details.
 
 [ "$DEBUG" == 1 ] && set -x
+set -u -e
 
 set -E # Enable function inheritance of traps
 trap exit ERR
+unset GETOPT_COMPATIBLE
+unsupported_getopt () {
+    echo "Your getopt(1) doesn't support quoting" >&2
+    exit 1
+}
+if getopt -T > /dev/null; then unsupported_getopt; elif [ "$?" -ne 4 ]; then unsupported_getopt; fi
```

Comment:
The intended use does not require any spaces in arguments, so I think this check is overzealous (I know **incorrect** use with no quoting would result in a weird error, but I don't think it's really relevant).